librz/analysis/var_global.c

```diff
@@ -0,0 +1,311 @@
+// SPDX-FileCopyrightText: 2021 Basstorm <basstorm@nyist.edu.cn>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_analysis.h>
+#include <rz_list.h>
+
+RZ_API RzAnalysisVarGlobal *rz_analysis_var_global_new(char *name, ut64 addr, char *comment) {
```

Comment:
Please don't forget to denote ownership with `RZ_OWN`/`RZ_BORROW`.

Reply:
and the RZ_NONNULL